lms/djangoapps/instructor/access.py

```diff
@@ -72,15 +75,22 @@ def _change_access(course, user, level, action, send_email=True):
 
     NOTE: will create a group if it does not yet exist.
     """
-
     try:
         role = ROLES[level](course.id)
     except KeyError:
         raise ValueError("unrecognized level '{}'".format(level))
 
     if action == 'allow':
         if level == 'ccx_coach':
-            email_params = get_email_params(course, True)
+            # Try to using CCX course name instead of the course.
+            try:
+                ccx_course_object = CustomCourseForEdX.objects.get(pk=course.id.ccx)
+                display_name = ccx_course_object.display_name
+            except CustomCourseForEdX.DoesNotExist:
+                display_name = None
```

Comment:
what will be shown in this case?

Reply:
could you provide an example?

Reply:
![screenshot from 2018-02-14 17-15-45](https://user-images.githubusercontent.com/25189966/36196412-8c8e6350-1168-11e8-8d52-bdb49cd73751.png)

if CustomCourseForEdX does not exist it will use master course name in this case is ` Master Course`
see, https://github.com/Livit/Livit.Learn.EdX/blob/labster/develop/lms/djangoapps/instructor/enrollment.py#L300